Dockerfile.build

```
@@ -6,6 +6,8 @@ ENV GOPATH=/go
 ENV PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
 
 COPY install-deps.sh /install-deps.sh
+COPY install_audio_deps.sh /install_audio_deps.sh
+
 RUN /install-deps.sh
```

```suggestion
RUN /install-deps.sh
RUN /install_audio_deps.sh
```

I need to add `RUN /install_audio_deps.sh` here in order to make the `./dev_deploy.sh` script work. Not sure if it is related to my dev env or needed in general.

---

Thanks for catching this! I am using devpod myself, so that's why I've missed it.

Thanks,
Alex